lab/README.md

```diff
@@ -0,0 +1,21 @@
+# Assignment Overview: Lab for Class 8
```

Comment:
Why are you committing the instructions from the assignment into your own repository?